Backend/app/rider-platform/rider-app/Main/src/API/Dashboard/RideBooking/Booking.hs

```diff
@@ -0,0 +1,67 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+module API.Dashboard.RideBooking.Booking where
+
+import qualified API.UI.Booking as UB
+import qualified Domain.Action.UI.Booking as DBooking
+import qualified Domain.Types.Booking as SRB
+import qualified Domain.Types.Booking.API as DB
+import qualified Domain.Types.Person as DP
+import Environment
+import EulerHS.Prelude
+import Kernel.Storage.Esqueleto
+import Kernel.Types.Id
+import Servant
+
+data RideBookingEndPoint
+  = RideStatusEndPoint
```

Comment:
Shouldn't prefix be Booking instead of Ride?

Reply:
@0utkarsh Booking is already their for , customer booking list, stuckBooking etc, it can create confusion so for rideBooking flow , i used rideBooking